### PR TITLE
Docs - Edit 

### PR DESCRIPTION
**Pull request by RakshithaK06**
Story details: https://app.clubhouse.io/kintohub/story/747

### CODE REVIEW

File: docs/anatomy/anatomy-environment.md [hunk 1]
```diff
@@ -25,6 +25,43 @@ This feature is only available for **Pay-As-You-Go** users. Free users are limit
 5. Create a name for your environment.
 6. Start adding services to your environment
 
+## Edit Environment
+
+1. Login to your account
+2. Click on the **Cogwheel** icon displayed at the top-right corner.
```
Comment (Disturbing) [inline docs/anatomy/anatomy-environment.md:31]: Action - settings icon.

File: docs/anatomy/anatomy-environment.md [hunk 2]
```diff
@@ -25,6 +25,43 @@ This feature is only available for **Pay-As-You-Go** users. Free users are limit
...
+
+:::note
+1. The **Region** field is locked and cannot be edited. 
+2. The **Environment ID** is a unique ID generated for each environment created and can be copied.
+:::
+
+## Delete Environment
+
+1. Login to your account
+2. Click on the **Cogwheel** icon displayed at the top-right corner.
```
Comment (Disturbing) [inline docs/anatomy/anatomy-environment.md:50]: Action - settings icon.

File: docs/anatomy/anatomy-environment.md [hunk 3]
```diff
@@ -25,6 +25,43 @@ This feature is only available for **Pay-As-You-Go** users. Free users are limit
...
+3. Edit your **Environment name** in the slide panel displayed.
+4. Click on the **Save** button
+
+On clicking `Save` button, your edited `Environment Name` will be saved.
+
+![Save Environment](/img/anatomy/edit-save.png)
+
+:::note
+1. The **Region** field is locked and cannot be edited. 
+2. The **Environment ID** is a unique ID generated for each environment created and can be copied.
```
Comment (Disturbing) [inline docs/anatomy/anatomy-environment.md:44]: action -  is a unique id for your environment and may be requested by the support team when aiding you with an issue.

File: docs/anatomy/anatomy-environment.md [hunk 4]
```diff
@@ -25,6 +25,43 @@ This feature is only available for **Pay-As-You-Go** users. Free users are limit
...
+3. Click on the **Delete Environment** button displayed in a slide panel.
+
+![Delete environment](/img/anatomy/delete-env.png)
+
+4. Click on the **Delete** button in the prompt displayed to delete your `Environment`.
+
+![Confirm Delete](/img/anatomy/confirm-del-env.png)
+
+:::important
+1. Environments cannot be deleted if you have your `services` running in it, all your services in a environment should be deleted before deleting a environment.
```
Comment (Disturbing) [inline docs/anatomy/anatomy-environment.md:60]: action - running in it. All services in your environment must be deleted before you can delete your environment.

File: docs/anatomy/anatomy-environment.md [hunk 5]
```diff
@@ -25,6 +25,43 @@ This feature is only available for **Pay-As-You-Go** users. Free users are limit
...
+2. The `Default` and `last` environment cannot be deleted.
```
Comment (Disturbing) [inline docs/anatomy/anatomy-environment.md:61]: action - use bullet instad of number

Comment (Disturbing) [reply docs/anatomy/anatomy-environment.md:61]: You must have at least 1 environment on your account. The deletion functionality is disabled if you only have 1 environment.

**Review by Disturbing — CHANGES_REQUESTED** [review]

File: docs/anatomy/anatomy-environment.md (hunk 1 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-environment.md:31]: Done

File: docs/anatomy/anatomy-environment.md (hunk 2 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-environment.md:50]: Done

File: docs/anatomy/anatomy-environment.md (hunk 3 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-environment.md:44]: Done

File: docs/anatomy/anatomy-environment.md (hunk 4 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-environment.md:60]: Done

File: docs/anatomy/anatomy-environment.md (hunk 5 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-environment.md:61]: Done

**Pull request merged**